lib/core/prometheusMetrics.hpp

```diff
@@ -16,6 +14,10 @@
 
 
 namespace kotekan {
+
+    class restServer;
+    class connectionInstance;
+
```

Comment:
The reason is that PrometheusMetrics.hpp and restServer.hpp are referencing each other, so I have to remove one include in one of them and add this forward declaration instead. Otherwise, there will be compiling error of undefined symbols.